src/components/common/buttons/CustomButton.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import React from 'react'
-import { View } from 'react-native'
+import { SafeAreaView, View } from 'react-native'
```

Comment:
it shouldn't be used here. add missing SafeAreaView to the screens where we have bug with this button